### PR TITLE
Created API and files to deploy to Heroku to test API

### PR DESCRIPTION
**Pull request by CodingPenguin**
PLS APPROVE LIKE A BOSS

### CODE REVIEW

File: sentysis/sentysis.py
Comment (leevanielle) [inline sentysis/sentysis.py:22]: LOL

File: sentysis/sentysis.py
Comment (leevanielle) [inline sentysis/sentysis.py:24]: You don't need `'GET'`

**Review by leevanielle — CHANGES_REQUESTED** [review]

File: sentysis/sentysis.py
Comment (CodingPenguin) [reply sentysis/sentysis.py:24]: Ok yah got it

**Comment by leevanielle** [comment]
Next time don't merge until I approve. Thanks!

**Pull request merged**
